src/components/leaderboard/LeaderboardRow/style.module.scss

```diff
@@ -0,0 +1,74 @@
+@use 'src/styles/vars.scss' as vars;
+
+.row {
+  align-items: center;
+  background-color: var(--theme-surface-1);
+  display: grid;
+  font-size: 15px;
+  gap: 32px;
+  grid-template-columns: auto auto 2fr 1fr;
+  height: 64px;
+  padding: 0 32px;
+
+  &[data-style='odd'] {
+    background-color: var(--theme-background);
+  }
+}
+
+.position {
+  font-size: 20px;
+  font-weight: bold;
+  width: 40px;
+}
+
+.profilePicture {
+  border-radius: 50%;
+}
+
+.nameRank {
+  display: grid;
+  gap: 32px;
+  grid-template-columns: 1fr 1fr;
+}
+
+.name,
+.points {
+  font-weight: 500;
+}
+
+.rank {
+  color: var(--theme-text-on-surface-1);
+  opacity: 0.5;
+  text-align: center;
+}
+
+.points {
+  text-align: right;
+}
+
+@media (max-width: 800px) {
```

Comment:
Nit: Let's make sure we're using our media breakpoints here defined in `vars.css` so breakpoints can be consistent across the website (so in the future the table will go into mobile mode the same time the navbar will). The vars you'll want to use are: 
`$breakpoint-sm: 425px;
$breakpoint-md: 768px; 
$breakpoint-lg: 1024px;`